src/backend/api/meals.js

```diff
@@ -2,11 +2,111 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
+//Returns all meals
 router.get("/", async (request, response) => {
+  let maxPrice = 100000;
+  let title = "%";
+  let createdAfter = 2000;
+  let limit = (await knex("meal")).length;
+  let availableReservations = false;
+  const keyArray = Object.keys(request.query);
```

Comment:
Nitpick(naming): Try to avoid the Hungarian notation, maybe `queryKeys` is more clear?